src/Service/CoverStore/CloudinaryTransformationService.php

```diff
@@ -47,6 +47,10 @@ public function transform(string $url, string $format = 'default'): string
 
         // Insert named transformation if it exists.
         if (!empty($transformation['transformation'])) {
+            // Check if the original image is large than the transformation size. If it is not return null.
+            if ($transformation['size'] > $height) {
```

Comment:
Should this also apply to width?

Reply:
Mainly to have it there if we need it later on...